features/step_definitions/brief_response_steps.rb

```diff
@@ -41,6 +41,19 @@
   }
 end
 
+Given 'that supplier has a completed brief response' do
```

Comment:
This step name makes it sound like the brief response has been submitted (which it is not) and completely changes the context. Some ideas of the top of my head
Given that supplier has filled in their application but not yet submitted